### PR TITLE
Switch to using hawk for authenticating

### PR DESCRIPTION
**Pull request by gregarndt**

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -52,8 +52,10 @@ let schema = Joi.object().keys({
     apiUrl: Joi.string().required().
       description('location of treeherder api (must end in /api/)'),
 
-    credentials: Joi.string().required().
-      description('entire treeherder/etl/data/credentials.json file')
+    credentials: Joi.object().required().keys({
```
Comment (selenamarie) [inline src/config.js]: Could you add a description back in?

File: src/jobs/treeherder_resultset.js [hunk 2]
```diff
@@ -36,22 +33,25 @@ export default class TreeherderResultsetJob extends Base {
...
-      consumerKey: cred.consumer_key,
-      consumerSecret: cred.consumer_secret,
+      clientId: this.config.treeherder.credentials.clientId,
+      secret: this.config.treeherder.credentials.secret,
       baseUrl: this.config.treeherder.apiUrl
     });
 
     let resultset = formatResultset(repo.alias, push);
-    await treeherderProject.postResultset([resultset]);
+    console.log(`Posting result set for project '${repo.alias}'`);
```
Comment (selenamarie) [inline src/jobs/treeherder_resultset.js]: yay

File: src/config.js (hunk 1 above)
Comment (gregarndt) [inline src/config.js]: addressed in abfa1031b3dc553926765923b3d4efbd872ba9e9

File: src/jobs/treeherder_resultset.js (hunk 2 above)
Comment (gregarndt) [inline src/jobs/treeherder_resultset.js]: logging makes me happy

**Pull request merged**